scene/register_scene_types.cpp

```diff
@@ -1085,6 +1085,10 @@ void unregister_scene_types() {
 	BaseMaterial3D::finish_shaders();
 #endif // _3D_DISABLED
 
+	PhysicalSkyMaterial::cleanup_shader();
+	PanoramaSkyMaterial::cleanup_shader();
+	ProceduralSkyMaterial::cleanup_shader();
```

Comment:
Should they be named `finish_shaders` to what other classes do?

Reply:
They dont do exacly the same I think